jumpscale/packages/admin/frontend/frontend_src/sources/services/packages.js

```diff
@@ -17,8 +17,8 @@ class PackagesService extends Service {
         return this.getCall("list_packages");
     }
 
-    packagesNames() {
-        return this.getCall("packages_names");
+    getInstalledPackages() {
```

Comment:
It should be the available packages no?

Reply:
No these are the installed ones only. The available packages are fetched by list_packages()